packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1717,19 +1778,29 @@ export default class Meeting extends StatelessWebexPlugin {
    * @memberof Meeting
    */
   closeLocalStream() {
-    return Media.stopTracks(this.mediaProperties.audioTrack)
-      .then(() => Media.stopTracks(this.mediaProperties.videoTrack))
+    const {audioTrack, videoTrack} = this.mediaProperties;
+
+    return Media.stopTracks(audioTrack)
+      .then(() => Media.stopTracks(videoTrack))
       .then(() => {
-        Trigger.trigger(
-          this,
-          {
-            file: 'meeting/index',
-            function: 'closeLocalStream'
-          },
-          EVENT_TRIGGERS.MEDIA_STOPPED, {
-            type: EVENT_TYPES.LOCAL
-          }
-        );
+        const audioStopped = audioTrack && audioTrack.readyState === ENDED;
+        const videoStopped = videoTrack && videoTrack.readyState === ENDED;
+
+        if (audioStopped && videoStopped) {
+          Trigger.trigger(
+            this,
+            {
+              file: 'meeting/index',
+              function: 'closeLocalStream'
+            },
+            EVENT_TRIGGERS.MEDIA_STOPPED, {
+              type: EVENT_TYPES.LOCAL
+            }
+          );
+        }
+        else if (audioTrack || videoTrack) {
+          LoggerProxy.logger.warn('Meeting:index#closeLocalStream --> Error: MediaStreamTrack.readyState is incorrect.');
```

Comment:
Should we print the readyState here

Reply:
I did at first but it resulted in two log statements (one for audio and video each), I figured one warning log statement per function is probably best to avoid too many logs. I can put it back.